src/legacy/containers/App.tsx

```diff
@@ -1,8 +1,18 @@
-import * as React from 'react'
+import React, { useEffect, useState } from 'react'
```

Comment:
Please do not change legacy code unless it's being moved to the features directory